src/server/Server.java

```diff
@@ -1,48 +1,45 @@
 package server;
 
-import cubyz.Logger;
+import cubyz.utils.Logger;
 import cubyz.client.ClientSettings;
 import cubyz.client.Cubyz;
 import cubyz.client.entity.ClientEntityManager;
+import cubyz.utils.Pacer;
 import cubyz.world.NormalChunk;
 
-public class Server {
+public class Server{
 	public static final int UPDATES_PER_SEC = 20;
 	public static final int UPDATES_TIME_NS = 1_000_000_000 / UPDATES_PER_SEC;
 	public static final float UPDATES_TIME_S = UPDATES_TIME_NS / 10e9f;
-	public static boolean running;
+
+	//set the pacer
+	public static final Pacer serverPacer = new Pacer("Server") {
```

Comment:
I think using `Server extends Pacer` would be cleaner than making an anonymous class here.

Reply:
+1